contrib/helm/Makefile

```diff
@@ -0,0 +1,44 @@
+# create kind cluster with nginx-ingress as the most popular ingress controller for K8S
+.PHONY: deploy
+create-cluster:
+	kind create cluster --name oauth2-proxy --config kind-cluster.yaml
+	make setup-dns
+	make setup-ingress
+
+.PHONY: setup-ingress
+setup-ingress:
+	kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/master/deploy/static/provider/kind/deploy.yaml
```

Comment:
Since this links to master, it's liable to change, should we be pinning it instead?

Reply:
I followed their example https://kind.sigs.k8s.io/docs/user/ingress/#ingress-nginx. It makes sense not to pin since we don't pin the version of kind. Installing the latest kind is expected to work with their documentation.